### DISCUSSION TITLE
REPOST: wire or square conductor path in 3 dimensions

### DISCUSSION THREAD

**Discussion by ringo458** [thliebig/openEMS-Project#33] [Q&A Usage]
Let me try this one again. I believe that I can probably generate a triangle mesh one way or another that describes the surface of a cylindrical conductor path in 3D. However this will not map so well onto the variable-pitch Cartesian mesh of this software. For many purposes at the frequency I'm interested in I could ignore the dimensions of the conductor but as it gets larger I have to start thinking about capacitance with adjacent, similar conductors. Since the path curve has compound curvature there is not a possibility to tweak the Cartesian mesh say, manually like you might have if it were straight in X for instance.

Fundamentally if I knew that I could ignore capacitance between parallel elements though I could concern myself with mutual inductance (which I can model in fasthenry) and call it a day. To that end I guess I'll do a model with two cylinders and manually tweak the mesh if I can figure out how to do that with openEMS. It's probably also OK to use a square cross section to first approximation anyway. Hollow, obviously.

**Comment by thliebig** [comment]
Well is there a question? This just sounds like an accurate description of a difficult simulation setup ;)

**Reply by ringo458** [reply]
It's enough to know that I haven't got it completely wrong :)
